recipes/mpfr/all/conandata.yml

```
@@ -1,4 +1,4 @@
 sources:
   "4.0.2":
     sha256: ae26cace63a498f07047a784cd3b0e4d010b44d2b193bab82af693de57a19a78
-    url: https://www.mpfr.org/mpfr-current/mpfr-4.0.2.tar.gz
+    url: https://ftp.gnu.org/gnu/mpfr/mpfr-4.0.2.tar.gz
```

```suggestion
    url: [
      "https://www.mpfr.org/mpfr-current/mpfr-4.0.2.tar.gz",
      "https://ftp.gnu.org/gnu/mpfr/mpfr-4.0.2.tar.gz",
    ]
```

Use gnu url as a mirror instead. I can download from mpfr, so probably it was a temporary error.

---

Are you sure?
```
$ wget https://www.mpfr.org/mpfr-current/mpfr-4.0.2.tar.gz
--2020-07-24 14:53:48--  https://www.mpfr.org/mpfr-current/mpfr-4.0.2.tar.gz
Resolving www.mpfr.org (www.mpfr.org)... 152.81.144.153
Connecting to www.mpfr.org (www.mpfr.org)|152.81.144.153|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2020-07-24 14:53:48 ERROR 404: Not Found.
```

---

My bad, I downloaded https://www.mpfr.org/mpfr-current/mpfr-4.1.0.tar.gz, the latest version available.